src/CLI/renderer/templates/show_interface_portchannel.j2

```diff
@@ -1,12 +1,53 @@
+{% set mode, minlinks, fallback = "lacp", "1", "false" %}
+
+
+{% if json_output['portchannel']['sonic-portchannel:LAG_TABLE'] and json_output['portchannel']['sonic-portchannel:LAG_TABLE']['LAG_TABLE_LIST'] %}
+{% for po_intf in json_output['portchannel']['sonic-portchannel:LAG_TABLE']['LAG_TABLE_LIST'] %}
+
+
+{% if json_output['portchannel']['sonic-portchannel:PORTCHANNEL'] and json_output['portchannel']['sonic-portchannel:PORTCHANNEL']['PORTCHANNEL_LIST'] %}
+{% for po_conf_intf in json_output['portchannel']['sonic-portchannel:PORTCHANNEL']['PORTCHANNEL_LIST'] %}
+
+{% if po_conf_intf['name'] == po_intf['lagname'] %}
+
+
+{% if po_conf_intf['name'] == po_intf['lagname'] and po_conf_intf['static'] == "true" %}
+{% set mode = "static" %}
+{% endif %}
+
+{% if po_conf_intf['name'] == po_intf['lagname'] and 'min_links' in po_conf_intf %}
+{% set minlinks = po_conf_intf['min_links'] %}
+{% endif %}
+
+
+{% if po_conf_intf['name'] == po_intf['lagname'] and 'fallback' in po_conf_intf %}
+{% set fallback = po_conf_intf['fallback']  %}
+{% endif %}
+
+{{po_intf['lagname']}} is {{po_intf['admin_status']}}, line protocol is {{po_intf['oper_status']}}
+mode {{mode}} min-links {{minlinks}} fallback {{fallback}}
```

Comment:
can please we make all attributes comma separated or in new line.

Reply:
Modified display strings